### ISSUE TITLE
Current releases of this and github.com/pkg/errors are not compatible

### ISSUE THREAD

**Issue by dcormier** [airbrake/gobrake#88]
Because of [this change](https://github.com/pkg/errors/commit/4f47277723cbe176eaef3bccb66a69de7a531157#diff-7dc02266d93e96cde6ba279eadbcac8fR12), `gobrake` v3.7.0 consumers also using `errors` v0.8.1 (the current releases of each) get this error:

```text
vendor/github.com/airbrake/gobrake/stack.go:79:28: cannot convert f (type "[snip]/vendor/github.com/pkg/errors".Frame) to type uintptr
```

**Comment by dcormier** [comment]
Looks like this was fixed by #87. It would be nice if a release were tagged so that `dep` users don't have to add a constraint to use `gobrake@master`.

**Comment by vmihailenco** [comment]
I've tagged v3.7.1
